.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "bracketSpacing": true,
+  "singleQuote": true,
+  "trailingComma": "all",
+  "printWidth": 100,
```

Comment:
why not keeping 120 like in the unrug frontend repo?

Reply:
Oh, didn't notice that frontend uses 120, will change it asap